hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestTableSnapshotScanner.java

```diff
@@ -460,7 +460,7 @@ public void testMergeRegion() throws Exception {
       }
     } catch (Exception e) {
       LOG.error("scan snapshot error", e);
-      Assert.fail("Should not throw FileNotFoundException");
+      Assert.fail("Should not throw Exception: " + e.getMessage());
```

Comment:
We have a fail call here, then the two asserts below are useless?

Reply:
Well spotted. Had removed these unreachable asserts.